base/abstractarray.jl

```diff
@@ -38,18 +38,32 @@ function first(itr)
 end
 last(a) = a[end]
 
+"""
+    stride(A, k)
+
+Returns the distance in memory (in number of elements) between adjacent elements in dimension `k`.
+"""
 function stride(a::AbstractArray, i::Integer)
-    if i > ndims(a)
-        return length(a)
-    end
+    i > ndims(a) && return length(a)
     s = 1
     for n=1:(i-1)
         s *= size(a, n)
     end
     return s
 end
 
-strides(a::AbstractArray) = ntuple(i->stride(a,i), ndims(a))::Dims
+"""
+    strides(A)
+
+Returns a tuple of the memory strides in each dimension.
+"""
```

Comment:
I think which method the docstring is attached to makes a difference, so maybe have it attached to the more general `strides(A::AbstractArray)` one?